docs/loading/Loading_data_template.md

```diff
@@ -0,0 +1,348 @@
+>
+> `ADMIN SET FRONTEND CONFIG ('default_replication_num' ="1");`
+
+```sql
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+
+CREATE TABLE `user_behavior_inferred` AS
+SELECT * FROM FILES(
+    "path" = "s3://starrocks-datasets/user_behavior_sample_data.parquet",
+    "format" = "parquet",
+    "aws.s3.region" = "us-west-1",
+    "aws.s3.access_key" = "AAAAAAAAAAAAAAAAAAAA",
+    "aws.s3.secret_key" = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+);
+```
+
+```SQL
+DESCRIBE user_behavior_inferred;
+```
+
+```plaintext
+Field       |Type            |Null|Key  |Default|Extra|
+------------+----------------+----+-----+-------+-----+
+UserID      |bigint          |YES |true |       |     |
+ItemID      |bigint          |YES |true |       |     |
+CategoryID  |bigint          |YES |true |       |     |
+BehaviorType|varchar(1048576)|YES |false|       |     |
+Timestamp   |varchar(1048576)|YES |false|       |     |
+```
+
+> Note
+>
+> Compare the inferred schema with the schema created by hand:
+>
+> - data types
+> - nullable
+> - key fields
+
+```SQL
+SELECT * from user_behavior_inferred LIMIT 10;
+```
+
+```plaintext
+UserID|ItemID|CategoryID|BehaviorType|Timestamp          |
+------+------+----------+------------+-------------------+
+171146| 68873|   3002561|pv          |2017-11-30 07:11:14|
+171146|146539|   4672807|pv          |2017-11-27 09:51:41|
+171146|146539|   4672807|pv          |2017-11-27 14:08:33|
+171146|214198|   1320293|pv          |2017-11-25 22:38:27|
+171146|260659|   4756105|pv          |2017-11-30 05:11:25|
+171146|267617|   4565874|pv          |2017-11-27 14:01:25|
+171146|329115|   2858794|pv          |2017-12-01 02:10:51|
+171146|458604|   1349561|pv          |2017-11-25 22:49:39|
+171146|458604|   1349561|pv          |2017-11-27 14:03:44|
+171146|478802|    541347|pv          |2017-12-02 04:52:39|
+```
+
+#### Loading into an existing table
+
+By querying the data directly from S3 it is possible to decide the:
+
+- column data types
+- column names
+- columns to use in the key
+
+```SQL
+CREATE TABLE `user_behavior_declared` (
+    `UserID` int(11),
+    `ItemID` int(11),
+    `CategoryID` int(11),
+    `BehaviorType` varchar(65533),
+    `Timestamp` datetime
+) ENGINE=OLAP 
+DUPLICATE KEY(`UserID`)
+DISTRIBUTED BY HASH(`UserID`)
+PROPERTIES (
+    "replication_num" = "1"
+);
+```
+
+After creating the table, you can load it with `INSERT INTO` … `SELECT FROM FILES()`:
+
+```SQL
+INSERT INTO user_behavior_declared
+  SELECT * FROM FILES(
+    "path" = "s3://starrocks-datasets/user_behavior_sample_data.parquet",
+    "format" = "parquet",
+    "aws.s3.region" = "us-west-1",
+    "aws.s3.access_key" = "AAAAAAAAAAAAAAAAAAAA",
+    "aws.s3.secret_key" = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+);
+```
+
+## More information
```

Comment:
Will be helpful to include a link to our BROKER LOAD SQL reference doc at https://docs.starrocks.io/en-us/main/sql-reference/sql-statements/data-manipulation/BROKER%20LOAD? Because we only cover some basic parameters in the load doc, and the user may need more info about the syntax and the parameters they can configure in their load job.